docs/source/torchaudio.rst

```diff
@@ -34,75 +33,3 @@ it easy to handle audio data.
    save
    save_with_torchcodec
    list_audio_backends
-
```

Comment:
`info` is still listed above but it was removed. Same with `list_audio_backends`. Let's follow-up in another PR

Reply:
Removed.